### PR TITLE
Updated contributing guidelines and commands

### LINKED ISSUE #114: コントリビューティングの修正と英語版作成
### 主題

コントリビューティングの修正と英語版作成

### 説明

コントリビューティングの修正と英語版を作成する

### この機能の作成に参加し、プルリクエストを作成しますか？

None

### PR DESCRIPTION
**Pull request by hirotomoyamada**
Closes #114

## Description

Updated contributing guidelines and commands.

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,3 +1,95 @@
...
+
+4. Run `pnpm dev` to start the local server.
+
+## Development
+
+To improve our development process, we have set up tools and systems.
+
+### Tooling
+
+- [Next.js](https://nextjs.org/) Framework for web sites.
```
Comment (taroj1205) [inline CONTRIBUTING.md:33]: Framework for the website?

Comment (hirotomoyamada) [reply CONTRIBUTING.md:33]: Fixed

**Review by taroj1205 — DISMISSED** [review]
LGTM

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,3 +1,95 @@
...
+
+4. Run `pnpm dev` to start the local server.
+
+## Development
+
+To improve our development process, we have set up tools and systems.
+
+### Tooling
+
+- [Next.js](https://nextjs.org/) Framework for website.
```
Comment (taroj1205) [inline CONTRIBUTING.md:33]: for "the" websiteです。

Comment (hirotomoyamada) [reply CONTRIBUTING.md:33]: Fixed

**Review by taroj1205 — APPROVED** [review]
Looks good to me!

**Pull request merged**